packages/core/src/wallet/Wallet.ts

```diff
@@ -9,6 +9,16 @@ import type {
 } from '../types'
 import type { Buffer } from '../utils/buffer'
 
+// Split up into WalletManager and Wallet instance
```

Comment:
This is a TODO?

Reply:
Ah damn. No i thought i wanted to do that now, but it's a biggie and we should move this to 0.6